.buildkite/scripts/grapl_testing_stack/destroy_stateful_resources.sh

```diff
@@ -49,22 +49,27 @@ mapfile -t target_args < <(echo "${urns}" | jq -r '. | map("--target=" + .) | .[
 
 echo -e "--- :pulumi: Destroying stateful resources for ${stack}"
 
-# Attempt to destroy each target, one at a time.
-# This can be de-for-loop-ified after
-# https://github.com/pulumi/pulumi/issues/3304
-for target_arg in "${target_args[@]}"; do
-    pulumi destroy \
-        --cwd="${project_dir}" \
-        --stack="${stack}" \
-        --show-replacement-steps \
-        --non-interactive \
-        --diff \
-        --yes \
-        --refresh \
-        --message="Destroying stateful resources" \
-        --target-dependents \
-        "${target_arg}" ||
-        true
-    # ^ Allow errors; each destroy is best-effort and should allow for
+(
```

Comment:
This is a relatively clean way to allow errors to only what's in the subshell command.

Reply:
thanks :)